packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -126,7 +126,7 @@ async function onMount() {
         nodeStatus.value = await getNodeStatus(nodeIDs.value);
         totalCost.value = getTotalCost(contracts.value);
         const TFTInUSD = await queryClient.tftPrice.get();
-        totalCostUSD.value = totalCost.value / TFTInUSD;
+        totalCostUSD.value = totalCost.value * TFTInUSD;
```

Comment:
this is correct, but we multiply the result by 1000 somewhere so the value is still wrong

Reply:
should be fixed